File: app/src/main/java/com/mutant/godutch/ViewHolderGroup.java
package com.mutant.godutch;

import android.support.v7.widget.AppCompatImageView;
import android.support.v7.widget.AppCompatTextView;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.LinearLayout;

/**
 * Created by evanfang102 on 2017/3/30.
 */

public class ViewHolderGroup extends RecyclerView.ViewHolder {

    public AppCompatImageView mImageViewPhoto;
    public AppCompatTextView mTextViewTitle;
    public AppCompatTextView mTextViewDescription;
    public AppCompatTextView mTextViewTotalPay;
    public LinearLayout mLinearLayoutFriends;

    public ViewHolderGroup(View itemView) {
        super(itemView);
        findViews(itemView);
    }

    private void findViews(View itemView) {
        mImageViewPhoto = (AppCompatImageView) itemView.findViewById(R.id.imageView_photo);
        mTextViewTitle = (AppCompatTextView) itemView.findViewById(R.id.textView_title);
        mTextViewDescription = (AppCompatTextView) itemView.findViewById(R.id.textView_description);
        mTextViewTotalPay = (AppCompatTextView) itemView.findViewById(R.id.textView_need_to_pay);
        mLinearLayoutFriends = (LinearLayout) itemView.findViewById(R.id.linearLayout_friends);
    }

}

File: app/src/main/java/com/mutant/godutch/ViewHolderGroup.kt
package com.mutant.godutch

import android.support.v7.widget.AppCompatImageView
import android.support.v7.widget.AppCompatTextView
import android.support.v7.widget.LinearLayoutCompat
import android.support.v7.widget.RecyclerView
import android.view.View
import kotlinx.android.synthetic.main.card_view_item_group.view.*

/**
 * Created by evanfang102 on 2017/3/30.
 */

class ViewHolderGroup(itemView: View) : RecyclerView.ViewHolder(itemView) {

    var mImageViewPhoto: AppCompatImageView = itemView.imageView_photo
    var mTextViewDate: AppCompatTextView = itemView.textView_date
    var mTextViewTitle: AppCompatTextView = itemView.textView_title
    var mTextViewTotal: AppCompatTextView = itemView.textView_my_total
    var mTextViewDebt: AppCompatTextView = itemView.textView_my_debt
    var mLinearLayoutFriends: LinearLayoutCompat = itemView.linearLayout_friends

}
